readme.md

```diff
@@ -95,13 +127,11 @@ To set up your local environment to run the unit tests:
 1. Open `wordpress-tests-lib/wp-tests-config.php` and add a slash to the end of the ABSPATH value. No clue why it leaves off the slash; it doesn't work without it.
 1. Run `phpunit` in the repo root dir. If you set `WP_CORE_DIR` and `WP_TESTS_DIR` above, you'll need to set them for this too. You should see output like this:
 
-    ```
```

Comment:
github's markdown renderer is pretty finicky and wants this exact syntax - four space indent, blank line before, and three backticks - to render as a code block _inside_ the list item. you'll probably need to preserve that exactly for it to still render right.